google/cloud/forseti/services/dao.py

```diff
@@ -489,7 +549,32 @@ def denorm_group_in_group(cls, session):
         @classmethod
         def explain_granted(cls, session, member_name, resource_type_name,
                             role, permission):
-            """Provide info about how the member has access to the resource."""
+            """Provide info about how the member has access to the resource.
+               For example, member m1 can access resource r1 with permission p
```

Comment:
Please add a newline here.

https://google.github.io/styleguide/pyguide.html#Comments

Reply:
I didn't get this one